charts/aspnetcore/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aspnetcore
 description: A generic Helm chart for ASP.NET Core services
-version: 1.0.4
+version: 1.0.5
```

Comment:
There is technically a breaking change but the removed field was undocumented and I couldn't find anyone making use of it.